src/uu/ln/src/ln.rs

```diff
@@ -163,11 +169,20 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
     let backup_mode = backup_control::determine_backup_mode(&matches)?;
     let backup_suffix = backup_control::determine_backup_suffix(&matches);
 
+    // When we have "-L" or "-L -P", false otherwise
+    let logical = if matches.is_present(options::LOGICAL) {
+        true
+    } else {
+        // When both are used Logical is stronger than Physical
+        matches.is_present(options::LOGICAL) && matches.is_present(options::PHYSICAL)
+    };
```

Comment:
I don't quite understand the logic here. If `matches.is_present(options::LOGICAL)` is `false` and we enter the `else` branch then `matches.is_present(options::LOGICAL) && matches.is_present(options::PHYSICAL)` must also be `false`, no? So this just becomes `let logical = matches.is_present(options::LOGICAL);` in practice.

Additionally, I think that Clap's `overrides_with` is symmetric, so since physical has `overrides_with(options::LOGICAL)`, the inverse is also true and we can never have both.

Reply:
From the clap docs on `overrides_with`:

> i.e. this argument and the following argument will override each other in POSIX style (whichever argument was specified at runtime last “wins”)

Reply:
Right, I removed the override, i should have removed it

Reply:
and you are right for the logic.